.changeset/warm-readers-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"jazz-tools": patch
+---
+
+Implemented SchemaUnion.Of()
```

Comment:
I'd like us to try to be more descriptive on the changelogs.
I know it's not easy, but if we work toghether should be more doable.

```suggestion
Added SchemaUnion, a new API that let's you assign multiple possible schemas to a given coValue and selectively apply them at runtime.
```